packages/core/src/interfaces/enums.ts

```diff
@@ -1,6 +1,11 @@
 import * as EventEnums from './events';
 export const Events = EventEnums;
 
+export enum RenderTypes {
+	HTML = 'html',
+	SVG = 'svg',
+}
+
```

Comment:
Just a heads up - two of the same exports of RenderTypes
1. packages/core/src/interfaces/enums.ts
2. packages/core/src/components/component.ts (line 12 - 16)